client/my-sites/comments/comment/comment-edit.jsx

```diff
@@ -137,6 +184,39 @@ export class CommentEdit extends Component {
 						/>
 					</FormFieldset>
 
+					<FormFieldset>
+						<FormLabel htmlFor="date">{ translate( 'Submitted on' ) }</FormLabel>
+						<Button
+							className="comment__edit-date-button"
+							ref={ this.storeDatePopoverButtonRef }
+							onClick={ this.toggleDatePopover }
+						>
+							{ moment( commentDate ).format( 'll LT' ) }
```

Comment:
I think this should be `lll` here otherwise we break the localization built-in to the formatter

Reply:
Nice catch!